scripts/DockerfileBackup

```diff
@@ -0,0 +1,65 @@
+FROM neotracker/neo-one-circleci-node:10.13.0-1 as builder
```

Comment:
I don't think we need a separate docker file for this - the credentials will just be provided in the environment that the docker image is running in.